src/Microsoft.TestPlatform.ObjectModel/Host/ITestRunTimeProvider.cs

```diff
@@ -126,14 +126,17 @@ public HostProviderEventArgs(string message)
             this.ErrroCode = 0;
         }
 
-        public HostProviderEventArgs(string message, int errorCode)
+        public HostProviderEventArgs(string message, int errorCode, int processId)
```

Comment:
Nit: 
Why don't we have separate cs file for these ?
Please add xml comments for anything that is public.